packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table/flatten-expanded-items-of-tree.ts

```diff
@@ -16,7 +16,7 @@ import { TableExpandStatus } from "../../entity-editor-context";
  * ```
  */
 export const flattenExpandedItemsOfTree = <
-  T extends { children: T[]; rowId: string },
+  T extends { children?: T[]; rowId?: string },
```

Comment:
Why are these changes necessary?

Reply:
Reverted this with 789280d087f815073f95c354daa52c80ee150bef.